client/client.go

```diff
@@ -244,10 +254,21 @@ func IsTwoFactorError(err error) bool {
 	return e.Kind == ErrorKindTwoFactorFailed || e.Kind == ErrorKindTwoFactorRequired
 }
 
+// OSRelease contains information about the system extracted from /etc/os-release.
+type OSRelease struct {
+	ID         string `json:"id"`
+	Name       string `json:"name"`
+	VersionID  string `json:"version-id,omitempty"`
+	Codename   string `json:"codename,omitempty"`
```

Comment:
Codename is an ubuntu-specific extension. AFAIR we still set it to "xenial" if it is missing. That shouldn't be happening.